_posts/0000-01-02-cmhume.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
“Ask me no questions, and I'll tell you no fibs.”
― Oliver Goldsmith, She Stoops to Conquer